src/components/Insights/common/JiraButton/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface JiraButtonProps {
+  handleTicketInfoButtonClick(spanCodeObjectId?: string): void;
+  ticketLink?: string | null;
+  spanCodeObjectId?: string;
+  buttonText?: string;
```

Comment:
I guess you can hardcode the button text and omit this prop as you already hardcode the menu width and `JiraButton` component should be used only for one purpose - Jira ticket